R/pkg/R/mllib.R

```diff
@@ -694,8 +694,8 @@ setMethod("predict", signature(object = "KMeansModel"),
 #' }
 #' @note spark.mlp since 2.1.0
 setMethod("spark.mlp", signature(data = "SparkDataFrame"),
-          function(data, blockSize = 128, layers = c(3, 5, 2), solver = "l-bfgs", maxIter = 100,
-                   tol = 0.5, stepSize = 1, seed = 1) {
+          function(data, blockSize = 128, layers, solver = "l-bfgs", maxIter = 100,
```

Comment:
Its also better to not make an argument required in the middle -- i.e. if we want to make layers a required argument then we should move it before blockSize